src/content/docs/en/pages/guides/devtools/customize-console-kit/index.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: How to customize the Azion Console using the Console Kit
+description: aaaa
+meta_tags: edge, console, development, interface, vue, primevue
+namespace: documentation_devtools_console_kit
+menu_namespace: devtoolsMenu
+permalink: /documentation/products/guides/customize-console-kit/
+---
+
+The Azion **Console Kit** allows you to craft a custom Azion interface to fit your business needs. In this guide, you'll learn how to make some simple visual changes to the Azion Console.
+
+:::caution[warning]
```

Comment:
Is it necessary to deploy the project to follow this guide? 

Coming from the CLI perspective:

- User will link the project to azion `azion link`
- The link commands adapts the project to azion
- The user can run it locally (no deployment required) 
- If the user wants to deploy he can do so, but it's optional.